clients/graphql/models.go

```diff
@@ -14,6 +15,26 @@ type GetAlertsResponse struct {
 	}
 }
 
+type IntermediateStruct struct {
+	Alerts struct {
+		PageInfo *PageInfo `json:"pageInfo"`
+		Alerts   []*struct {
+			Source SourceAlertEvent
+		} `json:"alerts"`
+	}
+}
+
+type SourceAlertEvent struct {
+	state         protoimpl.MessageState
```

Comment:
We can delete these unexported fields, the linter is right :slightly_smiling_face: